hexa/workspaces/models.py

```diff
@@ -0,0 +1,45 @@
+import uuid
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from hexa.core.models import Base
+
+
+class Workspace(Base):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.TextField()
+    description = models.TextField(blank=True)
```

Comment:
If `blank=True` you have to edit the graphql to reflect that it can be empty

Reply:
Yes, you right.